agent/util-scripts/gold/pbench-metadata-log/test-03.txt

```diff
@@ -1,9 +1,18 @@
 +++ Running test-03 pbench-metadata-log --dir=/var/tmp/pbench-test-utils/pbench/tmp beg
+/bin/ls: cannot access '/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/dhcp31-174/tools-default': No such file or directory
+/bin/ls: cannot access '/var/tmp/pbench-test-utils/pbench/tmp/pmdlog.NNNNN/perf104/tools-default': No such file or directory
 --- Finished test-03 pbench-metadata-log (status=0)
 +++ pbench tree state
 /var/tmp/pbench-test-utils/pbench
 /var/tmp/pbench-test-utils/pbench/tmp
 /var/tmp/pbench-test-utils/pbench/tmp/metadata.log
+/var/tmp/pbench-test-utils/pbench/tmp/pmdlog
```

Comment:
Where in the code is this `pmdlog` directory being created?  Seems like a bug.

Reply:
It was not: it was copied in from the sample directory (!?!). I'll post a fix in @riya-17's branch.